python/private/bzlmod/pip.bzl

```diff
@@ -323,6 +325,56 @@ def _create_whl_repos(module_ctx, pip_attr, whl_map, whl_overrides, group_map, s
 
     return is_hub_reproducible
 
+# TODO(groodt): Add more platforms
+_UV_DEPS = {
+    "linux_aarch64": (
+        "https://files.pythonhosted.org/packages/00/82/3d0acad7ebd4098aaa23f163b492b47fd9321b2681c55918fe5f18856513/uv-0.2.12-py3-none-manylinux_2_28_aarch64.whl",
+        "05d809516b651997a151585c9bbd150d888e71976a734781a3dde95430c1cab2",
+    ),
+    "linux_x86_64": (
+        "https://files.pythonhosted.org/packages/d9/85/e608bfe78772afa3f9807a074203982131ff9c12e279bb3bc502e14d51ab/uv-0.2.12-py3-none-manylinux_2_17_x86_64.manylinux2014_x86_64.whl",
+        "78d5329094de1f6b503bffe00abcdf1aaa7b3e4aa2aa373e128d7faf39b98de9",
+    ),
+    "osx_x86_64": (
+        "https://files.pythonhosted.org/packages/ff/f5/df24800e195f10e41aeb4a89f38488aca76c22d0f87c3eb8bf309f940aa3/uv-0.2.12-py3-none-macosx_10_12_x86_64.whl",
+        "85d2b9649c14014d6c54ecfb5d36f57e5c0e890957078ed4937e03569141d99b",
+    ),
+    "windows_x86_64": (
+        "https://files.pythonhosted.org/packages/c8/bc/a905f3d4b961781ead58243bd4f3c6cd2fcb2deaa55c565a64ba7d06cd20/uv-0.2.12-py3-none-win_amd64.whl",
+        "ca95af397769422e11e87dc147be1262a54d3a13d8837b69c80fad20f1ff9af4",
```

Comment:
I would prefer if we used `http_archive` with the URLs from GH: https://github.com/astral-sh/uv/releases

If we want to run it via Python in the future, we can do the wrapper ourselves. Usually our requirements are slightly different from regular python executions.

Reply:
Not a big change. We can make that happen in a non-spike version. What's the merit in doing this out of interest?

Reply:
It's a bit simpler overall. IIRC, the wheel is just a brief python program that calls subprocess() on prebuilt binary in the wheel. So it removes the Python naming scheme, the wheel format, and needing python to try and invoke it. Instead, you just get the tar.gz, unzip it, and have the uv binary directly.

It also makes it easier to use MODULE.bazel (or other first-class bazel-isms) to pick what version of uv you use and integrate with being a toolchain.